src/subscription_manager/managercli.py

```diff
@@ -1864,10 +1865,9 @@ def require_connection(self):
 
 class PluginsCommand(CliCommand):
     def __init__(self):
-        shortdesc = _("View and configure subscription-manager plugins")
+        shortdesc = _("View and configure %s plugins") % SM
```

Comment:
I'd suggest this to avoid bad translations for "plugins" since it is a module that the user would type at the command line...
shortdesc = _("View and configure %s %s") % (SM,"plugins")